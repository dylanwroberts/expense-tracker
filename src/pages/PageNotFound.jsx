import React from 'react';

const PageNotFound = () => (
    <div>
        Error: page not found
    </div>
);

export default PageNotFound;