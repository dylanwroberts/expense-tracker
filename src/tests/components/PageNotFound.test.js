import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../pages/PageNotFound';

test('should render the 404 page', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});