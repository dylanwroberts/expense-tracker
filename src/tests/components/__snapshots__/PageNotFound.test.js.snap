// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the 404 page 1`] = `
<div>
  Error: page not found
</div>
`;
